cpp/oneapi/dal/backend/primitives/reduction/reduction_dpc.hpp

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
```

Comment:
Naming is not aligned with the other primitives. `reduction_dpc.hpp` -> `reduction.hpp`.

Reply:
Fixed